tools/code_analyzer/run_analyzer.sh

```diff
@@ -14,10 +14,12 @@ echo "Analyze: ${INPUT}"
 # only _def and _impl are retained).  But the inliner isn't guaranteed
 # to operate, so for safety we match a more expansive set.
 "${ANALYZER_BIN}" \
-  -op_schema_pattern="^(_aten|_prim|aten|quantized|profiler|_test)::[^ ]+" \
-  -op_register_pattern="c10::RegisterOperators::(op|checkSchemaAndRegisterOp_)|c10::Module::(_?def|_?impl|impl_UNBOXED)" \
+  -op_schema_pattern="^(_aten|_prim|aten|quantized|profiler|_test)::[a-zA-Z0-9_.]+(\(.*)?$" \
+  -op_register_pattern="c10::RegisterOperators::(op|checkSchemaAndRegisterOp_)|c10::Module::(_?def|_?impl|impl_UNBOXED)|c10::Library::(_?def|_?impl|_?impl_UNBOXED)" \
   -op_invoke_pattern="c10::Dispatcher::findSchema|callOp" \
   -root_symbol_pattern="torch::jit::[^(]" \
+  -torch_library_init_pattern="^.*TORCH_LIBRARY_init_([^(]+)(\(.*)?$" \
+  -torch_library_init_pattern="^.*TORCH_LIBRARY_IMPL_init_([^(]+)_([^_]+)(\(.*)?$" \
```

Comment:
All hail the glory of backtracking. But I guess we will have to be careful to never define a Backend with an underscore in it...

Reply:
We can put some special delimiter when concatenating the init symbol name in the macro and update the RE.